tests/foreman/ui/test_discoveredhost.py

```diff
@@ -645,15 +656,17 @@ def test_positive_reboot(self):
             session.nav.go_to_select_org(self.org_name)
             with LibvirtGuest() as pxe_host:
                 hostname = pxe_host.guest_name
-                self._assertdiscoveredhost(hostname)
+                self.assertTrue(
+                    self.discoveredhosts.waitfordiscoveredhost(hostname)
+                )
                 element = (locators['discoveredhosts.fetch_ip'] % hostname)
                 # Get the IP of discovered host
                 host_ip = self.discoveredhosts.fetch_fact_value(
                     hostname, element)
                 # Check if host is reachable via IP
                 self.assertTrue(self._ping_host(host_ip))
                 self.discoveredhosts.reboot_host(hostname)
-                sleep(10)
+                sleep(15)
```

Comment:
Please try to apply something similar to https://gist.github.com/abalakh/7963c4304a5b2b2e68081215c2a59273 instead of regular sleep